x-pack/legacy/plugins/ml/public/application/datavisualizer/index_based/components/field_data_card/_field_data_card.scss

```diff
@@ -1,52 +1,94 @@
 .mlFieldDataCard {
-  height: 420px;
-  width: 360px;
+  height: 480px;
```

Comment:
This height is unnecessarily large. There is empty space below the content:

![image](https://user-images.githubusercontent.com/7405507/72254023-d5021100-35fa-11ea-88ba-49ab9b602f52.png)

`440px` I think would be sufficient to display the current content, although ideally we should aim to make the height no greater than the previous value of `420px`, if not smaller.

Reply:
cc @mdefazio